### PR TITLE
Use Font Files instead of google fonts url for better performance.

### PR DESCRIPTION
**Pull request by AbdoMgdy**

### CODE REVIEW

**Review by elhaw — COMMENTED** [review]
can we use three or four font files instead of adding them all?

**Review by MAlngaawy — COMMENTED** [review]
I deleted unused fonts files from the fonts folder and deleted its code from global.css. ( medium, thin )
it downed almost **from 290Kb To 140Kb**

File: frontend/styles/global.css [hunk 1]
```diff
@@ -51,3 +52,40 @@ p {
...
   height: 4px !important;
 }
+
+/* noto-sans-hk-regular - latin */
+@font-face {
+  font-family: 'Noto Sans HK';
+  font-style: normal;
+  font-weight: 400;
+  src: local('Noto Sans HK Regular'), local('NotoSansHK-Regular'),
+       url('../fonts/noto-sans-hk-v5-latin-regular.woff2') format('woff2'), /* Chrome 26+, Opera 23+, Firefox 39+ */
```
Comment (elhaw) [inline frontend/styles/global.css:62]: we can move the `fonts` folder inside the `public/static` path where we server all of our static files.
and change the path
Ex : ` url('../public/static/fonts/noto-sans-hk-v5-latin-regular.woff2') format('woff2'),`

**Review by elhaw — CHANGES_REQUESTED** [review]

**Comment by MAlngaawy** [comment]
I just changed the fonts file position to be in the public -> static file
and change the font-face URL directory at global.css

**Review by elhaw — APPROVED** [review]
LGTM

**Pull request merged**